p11-kit/proxy.c

```diff
@@ -248,7 +248,8 @@ modules_dup (CK_FUNCTION_LIST **modules)
 }
 
 static CK_RV
-proxy_create (Proxy **res, Mapping *mappings, unsigned int n_mappings)
+proxy_create (Proxy **res, CK_FUNCTION_LIST **loaded,
```

Comment:
I'm assuming this doesn't change API

Reply:
Yes, this shouldn't affect the API, although the semantics of `p11_kit_remote_serve_tokens()` becomes slightly different.  Previously it served only the first matching module, while it now serves all modules; I'd expect it's an acceptable change.